_common.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euxfo pipefail;
+
+find_images () {
+    docker images --format '{{.Repository}}:{{.Tag}}' | grep "^jackfirth/racket:[[:digit:]]" | sort
```

Comment:
Another small enhancement we could make is to change the scripts to read "$DOCKER_USERNAME" from the environment. That'd make it easier to test on others' (read: mine) computers.

Reply:
I went ahead and did this.